modules/core/04-channel/v2/keeper/msg_server_test.go

```diff
@@ -52,6 +52,7 @@ func (suite *KeeperTestSuite) TestRegisterCounterparty() {
 		{
 			"failure: channel must already exist",
 			func() {
+				suite.chainA.App.GetIBCKeeper().ChannelKeeperV2.DeleteCreator(suite.chainA.GetContext(), path.EndpointA.ChannelID)
```

Comment:
Not anymore, but it was needed to make the test fail in the first place. It passed before because the creator was still there (and it doesn't make a lot of sense for the creator to be present, but not the channel?).